src/lro/src/internal.rs

```diff
@@ -454,7 +454,7 @@ mod test {
         };
 
         use futures::StreamExt;
-        let mut stream = new_poller(
```

Comment:
seems unrelated?

Reply:
`clippy` started flagging this in 1.88.

`clippy` is right. We are calling it `mut`, but never mutating it.

Reply:
I guess it is unused mut warning